src/components/AllProducts/AllProducts.jsx

```diff
@@ -58,6 +59,7 @@ function AllProducts() {
         {allItemList.map((item) => (
```

Comment:
여기서 구조분해 할당해서 가져와도 좋겠네요!
```suggestion
        {allItemList.map(({id, images, name, price}) => (
```